src/welcome/services.ts

```diff
@@ -1,137 +1,62 @@
-        }
-
-        const contents = JSON.stringify(phrase);
-        fs.writeFileSync(await helpers.getKeyFileName(), contents);
-
-        // arweave
-        const arKey = getKeyFromMnemonic(phrase);
-        fs.writeFileSync(await helpers.getArweaveKeyFileName(), JSON.stringify(arKey));
-
-        // done
-        if (global.platform.win32) {
-            // const keyjson = '"C:\\Windows\\system32\\wsl.exe" cp "$("C:\\Windows\\system32\\wsl.exe" wslpath $("C:\\Windows\\system32\\wsl.exe" wslvar USERPROFILE))/.point/keystore/key.json" .';
-            // const arjson = '"C:\\Windows\\system32\\wsl.exe" cp "$("C:\\Windows\\system32\\wsl.exe" wslpath $("C:\\Windows\\system32\\wsl.exe" wslvar USERPROFILE))/.point/keystore/arweave.json" .';
-            const wslvar = 'wsl.exe wslvar USERPROFILE';
-            const keyjson = 'wsl.exe cp "$(wsl.exe wslpath $(wsl.exe wslvar USERPROFILE))/.point/keystore/key.json" ~/.point/keystore/key.json';
-            const arjson = 'wsl.exe cp "$(wsl.exe wslpath $(wsl.exe wslvar USERPROFILE))/.point/keystore/key.json" ~/.point/keystore/key.json';
-
-            await new Promise<void>((resolve, reject) => {
-                try {
-                    console.log({keyjson, arjson});
-                    console.log({wslvar});
-
-                    // key.json
-                    exec(wslvar, (error: { message: any; }, wslvarRes: string, stderr: string | undefined) => {
-                        if (error) {
-                            console.log(`error: ${error.message}`);
-                            return reject(error);
-                        }
-                        if (stderr) {
-                            console.log(`stderr: ${stderr}`);
-                            return reject(new Error(stderr));
-                        }
-                        wslvarRes = wslvarRes.trim();
-                        const wslPath = `wsl.exe wslpath "${wslvarRes}"`;
-                        console.log({wslPath});
+  async start() {
+    // ?
+  }
 
-                        exec(wslPath, (error: { message: any; }, wslPathRes: string, stderr: string | undefined) => {
-                            if (error) {
-                                console.log(`error: ${error.message}`);
-                                return reject(error);
-                            }
-                            if (stderr) {
-                                console.log(`stderr: ${stderr}`);
-                                return reject(new Error(stderr));
-                            }
-                            wslPathRes = wslPathRes.trim();
-                            const wslCP = `wsl.exe cp -f "${wslPathRes}/.point/keystore/key.json" ~/.point/keystore/`;
+  async login(phrase: any, firstTime = false) {
+    if (await helpers.isLoggedIn())
+      throw Error(
+        'Already logged in (~/.point/keystore/key.json already exists). You need to log out first.'
```

Comment:
How does the user log out? Is that part of the UI or does the user need to remove their `key.json` to log out?